### PR TITLE
AFL summary script

### PR DESCRIPTION
**Pull request by BramVerb**
Added a script to collect the error codes from AFL.

This allows students to easily view how many unique errors AFL finds.

### CODE REVIEW

File: docs/fuzzing_rers.md [hunk 1]
```diff
@@ -639,5 +641,19 @@ Abort trap: 6
 
 **Note**: In the newer version of RERS problems, you will not see `Abort trap: 6` but you will see `Aborted` instead.
 
+We have create a python script to get a summary of all the errors AFL finds. The script is located in `scripts/analyze_afl.py`. The script needs the afl findings directory and the binary to run the crash files with: `./scripts/analyze_afl AFL_FINDINGS_DIRECTORY PROBLEM_BINARY`. If you have the files and folders in line with the examples above, you can run `./scripts/analyze_afl findings Problem11` to get all the errors.
```
Comment (ClintonCao) [inline docs/fuzzing_rers.md]: Can you the following changes to selected sentences?:
`We have create a python script to get a summary of all the errors AFL finds.` --> `We created a Python script that computes the summary of the errors that were found by AFL`

`The script needs the afl findings directory and the binary to run the crash files with:` --> `The script requires the AFL finding directory and the binary to run the crash files with:`

**Review by ClintonCao — CHANGES_REQUESTED** [review]
Looks good, can you change the sentences based on the comments that I have left?

File: docs/fuzzing_rers.md (hunk 1 above)
Comment (BramVerb) [reply docs/fuzzing_rers.md]: Done!

Comment (ClintonCao) [reply docs/fuzzing_rers.md]: Thanks!

**Pull request merged**